### PR TITLE
Added Missing Shops Script

### PR DESCRIPTION
**Pull request by Pokye**
* **Addressed Issue(s)**: https://github.com/rathena/rathena/issues/6990

* **Server Mode**: Renewal

* **Description of Pull Request**: Still need to check the translation.

Thanks to @Balferian for discovering it, and to @Everade for helping me with a translation

### CODE REVIEW

File: npc/re/merchants/Emperium_Seller.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+//===== Description: ========================================= 
+//= Sell Emperium
+//===== Additional Comments: ================================= 
+//= 1.0 Special thanks for Balfear and Everade
+//============================================================ 
+
+prt_in,212,169,3	script	Guild Clerk	4_M_04,{
+	if (gettime(DT_DAYOFWEEK) >= 1 && gettime(DT_DAYOFWEEK) <= 6 && gettime(DT_HOUR) >= 18 && gettime(DT_HOUR) <= 23) {
+		mes "[Guild Clerk]";
+		mes "The Empelium to be distributed to the guild office today has arrived. If you want to purchase, please put as many as you want~";
```
Comment (secretdataz) [inline npc/re/merchants/Emperium_Seller.txt:18]: Emperium

File: npc/re/merchants/Emperium_Seller.txt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		next;
+		mes "[Guild Clerk]";
+		mes "It seems that many people are looking for Empelium today. It's just a personal feeling.";
```
Comment (secretdataz) [inline npc/re/merchants/Emperium_Seller.txt:21]: Emperium

File: npc/re/merchants/Emperium_Seller.txt [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	mes "[Guild Clerk]";
+	mes "Our guild office actively supports the smooth gathering of adventurers";
+	next;
+	mes "[Guild Clerk]";
+	mes "as part of that ^0000ffMonday-Saturday^000000";
+	mes "<ITEM>Emperium<INFO>714</INFO></ITEM> in a limited quantity of 100 pieces per day";
+	mes "Afternoon ^0000ff18:00~23:59^000000 is on sale.";
+	next;
+	mes "[Guild Clerk]";
+	mes "Please note that our guild office does not sell because we have to run around to supply Empelium outside of the hours.";
```
Comment (secretdataz) [inline npc/re/merchants/Emperium_Seller.txt:35]: Emperium

**Review by secretdataz — CHANGES_REQUESTED** [review]

File: npc/re/merchants/Emperium_Seller.txt [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    	callshop "emp_seller#slo";
+		end;
+    }
+	mes "[Guild Clerk]";
+	mes "Our guild office actively supports the smooth gathering of adventurers";
+	next;
+	mes "[Guild Clerk]";
+	mes "as part of that ^0000ffMonday-Saturday^000000";
+	mes "<ITEM>Emperium<INFO>714</INFO></ITEM> in a limited quantity of 100 pieces per day";
+	mes "Afternoon ^0000ff18:00~23:59^000000 is on sale.";
```
Comment (Atemo) [inline npc/re/merchants/Emperium_Seller.txt:32]: Maybe
```suggestion
	mes "As part of that,";
	mes "we are selling a limited quantity of 100 <ITEM>Emperium<INFO>714</INFO></ITEM> every day";
	mes "from ^0000ff18:00 to 23:59^000000 from ^0000ffMonday to Saturday^000000.";
```
is a bit better

File: npc/re/merchants/Emperium_Seller.txt [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+	mes "As part of that,";
+	mes "we are selling a limited quantity of 100 <ITEM>Emperium<INFO>714</INFO></ITEM> every day";
+	mes "from ^0000ff18:00 to 23:59^000000 from ^0000ffMonday to Saturday^000000.";
+	next;
+	mes "[Guild Clerk]";
+	mes "Please note that our guild office does not sell because we have to run around to supply Emperium outside of the hours.";
+	close;
+
+OnClock0000:
+	if (gettime(DT_DAYOFWEEK) == 1) npcshopupdate "emp_seller#slo",714,0,100;
```
Comment (Atemo) [inline npc/re/merchants/Emperium_Seller.txt:39]: According to the translation, shouldn't the shop be updated every day?
```suggestion
OnHour00:
	npcshopupdate "emp_seller#slo",714,0,100;
```

File: npc/re/merchants/Slot_Move_Card_Sales.txt [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+	next;
+	mes "[Salesman]";
+	mes "I sell it in a limited quantity every week, so if you don't see anything,";
+	mes "it means all of the items are sold out, so please come back next time.";
+	close2;
+	callshop "change_slot#slo";
+	end;
+
+OnClock0000:
+	if (gettime(DT_DAYOFWEEK) == 1) npcshopupdate "change_slot#slo",12786,0,9999;
```
Comment (Atemo) [inline npc/re/merchants/Slot_Move_Card_Sales.txt:29]:
```suggestion
OnMon0000:
	npcshopupdate "change_slot#slo",12786,0,9999;
```

File: npc/re/merchants/Emperium_Seller.txt [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+	if (gettime(DT_DAYOFWEEK) >= 1 && gettime(DT_DAYOFWEEK) <= 6 && gettime(DT_HOUR) >= 18 && gettime(DT_HOUR) <= 23) {
+		mes "[Guild Clerk]";
+		mes "The Emperium to be distributed to the guild office today has arrived. If you want to purchase, please put as many as you want~";
+		next;
+		mes "[Guild Clerk]";
+		mes "It seems that many people are looking for Emperium today. It's just a personal feeling.";
+		close2;                                                                                                         
+    	callshop "emp_seller#slo";
+		end;
+    }
```
Comment (Atemo) [inline npc/re/merchants/Emperium_Seller.txt:25]:
```suggestion
	}
```

File: npc/re/merchants/Emperium_Seller.txt [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+prt_in,212,169,3	script	Guild Clerk	4_M_04,{
+	if (gettime(DT_DAYOFWEEK) >= 1 && gettime(DT_DAYOFWEEK) <= 6 && gettime(DT_HOUR) >= 18 && gettime(DT_HOUR) <= 23) {
+		mes "[Guild Clerk]";
+		mes "The Emperium to be distributed to the guild office today has arrived. If you want to purchase, please put as many as you want~";
+		next;
+		mes "[Guild Clerk]";
+		mes "It seems that many people are looking for Emperium today. It's just a personal feeling.";
+		close2;                                                                                                         
+    	callshop "emp_seller#slo";
```
Comment (Atemo) [inline npc/re/merchants/Emperium_Seller.txt:23]:
```suggestion
		callshop "emp_seller#slo";
```

**Review by Atemo — APPROVED** [review]

**Review by aleos89 — APPROVED** [review]

**Pull request merged**